new_gym.py

```diff
@@ -118,6 +127,10 @@ def __init__(self, goal_pose=None, render=False):
                     0,
                     0,
                     0,
+                    0, 
+                    0, 
+                    0, 
+                    0, 
```

Comment:
Network size changed, please update the training agent demo script and test script as well